docs/Installation.md

```diff
@@ -1,68 +1,66 @@
 # Installation
-Follow this guide if you want to install this package in new\existing laravel installation.
+Follow this guide to install this package in a new or existing laravel project.
 
-Follow next steps one after another:
 1. [Laravel](#laravel)
 2. [Doctrine Setup](#doctrine-setup)
-3. [Package config](#package-config)
-4. [Verify installation](#verify-installation)
+3. [Package Installation](#package-install)
+4. [Package Config](#package-config)
+5. [Verify installation](#verify-installation)
```

Comment:
Please remove this line as we don't have this section anymore.
I think we can skip the section at all.

Reply:
Fixed